pkgs/top-level/all-packages.nix

```diff
@@ -19641,6 +19643,8 @@ with pkgs;
 
   vym = qt5.callPackage ../applications/misc/vym { };
 
+  wavepack = callPackage ../development/libraries/WavePack { };
```

Comment:
```suggestion
  wavpack = callPackage ../development/libraries/wavpack { };
```

Reply:
Dropped this commit: already in nixpkgs with the correct name.